pkg/reconciler/instances/istio/action.go

```diff
@@ -201,6 +201,16 @@ func canUninstall(istioVersion actions.IstioVersion) bool {
 	return isInstalled(istioVersion) && istioVersion.ClientVersion != ""
 }
 
+func (h *helperVersion) isDowngradePermitted(second *helperVersion) bool {
+	// we permit one minor downgrade, please see: https://github.com/kyma-incubator/reconciler/issues/365
+	if h.major == second.major {
+		if second.minor-h.minor == 0 || second.minor-h.minor == 1 {
+			return true
+		}
+	}
+	return false
+}
+
```

Comment:
Maybe also add here a function isUpdgradePermitted to have the logic compressed in the `helperVersion`